core/src/main/java/io/paradoxical/francois/ServiceApplication.java

```diff
@@ -108,7 +108,7 @@ private SwaggerConfiguration getPublicSwagger(final Environment environment) {
                 setContact("admin@francois.io");
                 setPrettyPrint(true);
 
-                setBasePath(environment.getApplicationContext().getContextPath());
+                setScan(true);
```

Comment:
I think this was a programming by coincidence type things. I reverted the change and things still work.  I guess scan sets the scanner singleton vs us doing isolated scans per resource endpoint right?